proto/frequenz/api/weather/weather.proto

```diff
@@ -69,28 +69,27 @@ enum ForecastFeature {
   FORECAST_FEATURE_SURFACE_NET_SOLAR_RADIATION = 4;
 }
 
-// The ForecastRequest message is used to specify the parameters for the
-// SubscribeWeatherForecast method. It allows users to request weather forecasts
-// for a specific location and time period, with specified features. 
-message ForecastRequest {
+// The GetHistoricalWeatherForecastRequest message defines parameters for
+// retrieving historical weather forecasts, targeting a specific location
+// and time period, with designated features.
+message GetHistoricalWeatherForecastRequest {
   // The location for which the forecast is being requested.
   frequenz.api.common.location.Location location = 1;
 
   // List of required features. If none are specified, all available features
-  // will be streamed.
+  // will be returned.
   repeated ForecastFeature features = 2;
 
-  // Optional for StreamLiveWeatherForecast, ignored if provided.
-  // The start of the period for which the forecast is being requested.
+  // The start of the period for which the historical forecast is being
+  // requested.
   google.protobuf.Timestamp start_ts = 3;
 
-  // Optional for StreamLiveWeatherForecast, ignored if provided.
-  // The end of the period for which the forecast is being requested.
+  // The end of the period for which the historical forecast is being
+  // requested.
   google.protobuf.Timestamp end_ts = 4;
 
-  // Optional for StreamLiveWeatherForecast, ignored if provided.
   // Specifies the maximum number of forecasts to be returned in a single
-  // response.
+  // response. If not provided, a default page size will be used.
```

Comment:
Looking at the final response messages in #25, the API would always respond with one forecast for multiple locations. So is this referring to the number of locations that will be in one forecast? If so, then I would adjust the docs here to make this more clear @thomas-nicolai-frequenz

Reply:
> the API would always respond with one forecast for multiple locations

This should be fixed in https://github.com/frequenz-floss/frequenz-api-weather/issues/19

However, the page_size parameter is not correct yet. I'll create another issue for us. Sorry that was my bad.

Reply:
Alright I also addressed the page_size issue in https://github.com/frequenz-floss/frequenz-api-weather/issues/19

Reply:
I suggest to remove the page_size from this PR and merge it before moving on to #19

Reply:
Ok, Then I will adjust pagination in PR #25 which fixes #19. 
Adjusted this PR to make no adjustments to page_size, except removing the comment that it is optional. Or did you mean I should delete the page_size field in this PR? 
Otherwise, this is ready for review.